app/sass/public.scss

```diff
@@ -60,4 +60,25 @@
 	margin-top: 12px;
 	text-align: center;
 	font-size: 1.2em;
-}
\ No newline at end of file
+}
+
+////////////////////////////////////////////////////////////////////////////////
+// .public-footer
+////////////////////////////////////////////////////////////////////////////////
+
+.public-footer {
+	display: flex;
+	justify-content: center;
+	width: 100%;
+	margin-top: 40px;
+
+	.country-flag:first-of-type {
```

Comment:
By default we dont use Sass _nesting_ in our `.scss`files.
Can you remove the nesting from here, please? :)

Reply:
Done!